RELEASING.md

```diff
@@ -19,9 +24,13 @@
   - make sure the PR title starts with `rel:` to conform with our semantic commits convention
   - add the `no-changelog`, `type: maintenance`, and `version: no bump` labels
 - Once the pull request is approved, squash-merge it
+
+## Git Tags and Releases
 - Checkout `main` and fetch the now-updated `main` branch.
-- Apply a tag for the new version on the merged commit (e.g. `git tag -a v2.3.1 -m "v2.3.1"`)
-- Push the tag upstream (this will kick off the release pipeline in CI) e.g. `git push origin v2.3.1`
+- Apply a tag for the new version on the merged commit (e.g. `git tag -a web-distro-v2.3.1 -m "web-distro-v2.3.1"`). The tag MUST begin with either `web-distro-v` or `react-instrumentation-v` in order to trigger our CI release pipeline
+  - use the tag prefix that corresponds to the library you want to release
```

Comment:
Should we have a table of tags OR make a script package `npm run release-tag:all` or `npm run release-tag:otel-web-vX.Y.Z` for tagging and pushing? The script tag will document the tag names and make it less error prone since it's now more involved?

Reply:
Automating more of this release process can be done in a later PR. For this PR, the important part is just getting to a multi-package world.

Reply:
Agree 💯 I'm lazy and don't want to typo a lot...so I'll followup with some scripts....eventually!